tox.ini

```diff
@@ -23,6 +24,13 @@ deps =
 commands =
     flake8 {posargs:.}
 
+[testenv:lint_yaml]
```

Comment:
I'd rather see this as just another command in the `lint` environment rather than in its own environment (and, if we have a format step, it'd just be a another command in the `format` environment).